.github/workflows/int_tests_cypress.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: cypress-io/github-action@v5
         with:
           working-directory: ./ui
-          config: video=false,videoUploadOnPasses=false,keystrokeDelay=1,retries=2
+          config: '{"e2e": { "excludeSpecPattern": ["*.hot-update.js", "**/*.e2e.cy.js"] }, "video": false, "keystrokeDelay": 1, "retries": 2,  "component": {"excludeSpecPattern": ["*.hot-update.js", "**/*.e2e.cy.js"]}}'
```

Comment:
Couldn't get it to work as before (inline config).. had to repalce with JSON. We'll clean up if we find an alternative, but this was blocking this PR.